site/disk-alarms.xml

```diff
@@ -83,12 +83,18 @@ Disabling disk free space monitoring</pre>
         required to be free on the database partition. (See the description of
         <a href="/relocate.html">file locations</a> for the default location).
         This configuration file sets the disk free space limit to 1GB:
+        <pre class="sourcecode">disk_free_limit.absolute = 1000000000</pre>
+        Or you can use memory units (KB, MB GB etc.) like this:
```

Comment:
[We do support `kB`](https://github.com/rabbitmq/rabbitmq-server/blob/master/src/rabbit_resource_monitor_misc.erl#L43).